main/survey/copy_survey.php

```diff
@@ -31,16 +31,18 @@
     // Clear token
     Security::clear_token();
     $surveyId = intval($_POST['surveys']);
-    $courseId = Security::remove_XSS($_POST['destination_course']);
-    $surveyCopyId = SurveyManager::copy_survey($surveyId, null, $courseId);
+    $arraysent = json_decode(Security::remove_XSS($_POST['destination_course']));
+    $courseId = $arraysent->id_course;
+    $sessionId = $arraysent->id_session;
+    $surveyCopyId = SurveyManager::copySurveySession($surveyId, $courseId, $sessionId);
     // Copy the survey to the target course
-    SurveyManager::empty_survey($surveyCopyId, $courseId);
+    SurveyManager::emptySurveyFromId($surveyCopyId);
```

Comment:
`// Copy the survey to the target course
    $surveyCopyId = SurveyManager::copySurveySession($surveyId, $courseId, $sessionId);
    // Empty the copied survey
    SurveyManager::emptySurveyFromId($surveyCopyId);`